File: ATMApplication/ATMAPI/Models/BankAccount.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace ATMAPI.Models
{
    public class BankAccount
    {
        [Key]
        public int Id { get; set; }
        [Required]
        public  string FullName { get; set; }
        [Required]
        public long AccountNumber { get; set; }
        [Required]
        public long CardNumber { get; set; }

        [Required]
        public decimal Balance { get; set; }

    }
}

File: ATMFrontEnd/ATMApplication/src/app/core/model/bank-account.ts
export class BankAccount {
    id: number;
    fullName: string;
    accountNumber: number;
    cardNumber: number;
    balance:number
    constructor( id: number,
        fullName: string,
        accountNumber: number,
        cardNumber: number,
        balance:number) {
         this.id = id;
         this.fullName = fullName;
         this.accountNumber = accountNumber;
         this.cardNumber =  cardNumber;
         this.balance = balance;
    }
}